src/flow-hash.c

```diff
@@ -107,6 +107,61 @@ typedef struct FlowHashKey6_ {
     };
 } FlowHashKey6;
 
+uint32_t FlowGetFtpHash(const Packet *p)
```

Comment:
this is not specific to ftp, right? Its essentially ippair+ipproto? If so lets reflect that in the name

Reply:
You are right (named it because it was ftp-friendly)